setup.py

```diff
@@ -134,10 +134,13 @@ def do_run(self, builder: str, output_dir: str):
     scripts=glob(join('bin', '*')),
     packages=find_namespace_packages(include=["cylc.*"]),
     package_data={
-        'cylc.flow': ['*.txt', '*.md', '*.sh']
+        'cylc.flow': [
+            'etc/*.yaml', 'etc/global*.eg', 'etc/job.sh',
+            'etc/syntax/*', 'etc/syntax/*/*',
```

Comment:
Can remove the developer files in `etc/syntax/*/*`?

Reply:
Good spotting; done.